scripts/local/utils/bulk_ops_common.py

```diff
@@ -107,15 +137,44 @@ def add_common_args(argparser, arguments):
 
 
 def run_argparser(argparser):
-    (
-        args,
-        unknown,
-    ) = argparser.parse_known_args()
-
+    """
+    Runs the argument parser and returns parsed arguments.
+    """
+    args, unknown = argparser.parse_known_args()
     args = vars(args)
 
-    if len(unknown) > 0:
+    if unknown:
         argparser.print_help()
         raise Exception(f"\nError: Unknown arguments: {unknown}")
 
     return args
+
+
+def main():
+    """
+    Main entry point for the script. Handles argument parsing and starts the bulk conversion process.
+    """
+    parser = argparse.ArgumentParser(description="Bulk image and PDF converter")
+
+    # Add standard arguments
+    add_common_args(parser, ['-i', '--input', '-o', '--output', '--recursive', '--trigger-size'])
+
+    parser.add_argument(
+        "--format", 
```

Comment:
Let's make this a bit more specific
```suggestion
        "--out-format", 
```